cli/command/container/stats.go

```diff
@@ -31,7 +31,7 @@ func NewStatsCommand(dockerCli *command.DockerCli) *cobra.Command {
 	var opts statsOptions
 
 	cmd := &cobra.Command{
-		Use:   "stats [OPTIONS] [CONTAINER...]",
+		Use:   "stats [OPTIONS] CONTAINER [CONTAINER...]",
 		Short: "Display a live stream of container(s) resource usage statistics",
 		Args:  cli.RequiresMinArgs(0),
```

Comment:
Sorry, I don't understand it just now.
If it don't need to add option "--format" in document， I will close this PR.